community/saas-companies-engagement-procedure.md

```diff
@@ -0,0 +1,118 @@
+## Procedure for Engaging With SaaS Companies
+
+### Summary
+
+This document outlines the procedure for engaging with SaaS companies, and
+how SaaS companies can represent themselves and their interests in the SPIFFE
+community.
+
+The goal is to provide a clear and consistent process for engaging with 
+SaaS companies that are included but not limited to those that use SPIFFE and 
+SPIRE, provide a proprietary SPIFFE-compatible service, or are interested in 
+contributing to the SPIFFE project.
+
+More specifically, this document includes guidelines for communication, 
+technical contact designation, and handling potential disruptive participants.
+
+### Technical Contacts
+
+#### Objective
+
+To ensure effective communication and support for SaaS consumers 
+using the SPIFFE specification by designating responsible technical contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+Each SaaS company that directs customers to the SPIFFE Slack channel must 
+designate two technical contacts.
+
+These contacts will serve as the primary point of contact for addressing queries 
+and facilitating communication between the customers and the Spiffy community.
```

Comment:
```suggestion
and facilitating communication between the customers and the SPIFFE community.
```

Reply:
And provide a mechanism for the SPIFFE community to communicate with the Company?

Reply:
There are several of those Spiffy’s. side effect of multitasking :) — will fix them all.